### DISCUSSION TITLE
Running from Subdirectories

### DISCUSSION THREAD

**Discussion by obustosm** [orchidsoftware/platform#2741] [Q&A]
Hi There!

I was using AdminLTE before reaching Orchid on Laravel.

Had the same exact trouble when using it from subdirectories:
![image](https://github.com/orchidsoftware/platform/assets/11504405/b8ef577d-a59c-4cf8-b264-63c703b45a72)
But modifying the layout file (app.blade.php) on this code block:
`
    @if(\Orchid\Support\Locale::currentDir(app()->getLocale()) == "rtl")
        <link rel="stylesheet" type="text/css" href="{{  mix('/css/orchid.rtl.css','vendor/orchid') }}">
    @else
        <link rel="stylesheet" type="text/css" href="{{  mix('/css/orchid.css','vendor/orchid') }}">
    @endif
`

Adding `{{ url('/') }}` inside ***href*** tags, before **MIX** blade tag resolves this behaviour on all subsequent sites/webpages.

The final questions are: 

- Is this the recommended solution?
- Is there any option that i can use on `platform.php`? Because editing `vendor/orchid/platform/resources/views/app.blade.php` isn't standard for saving it to my github project...

**Comment by macit-emre** [comment]
Not a legid answer buy you may override app.blade.php inside resources/views/vendor/platform/app.blade.php that you may commit it.

**Comment by nmalinoski** [comment]
The solution I found involves setting the `app.asset_url` and [`app.mix_url`](https://github.com/orgs/orchidsoftware/discussions/2377#discussioncomment-7701337) configuration items and the Vite `base` directory:

1. Set the following in `.env`:
    ```
    APP_URL="https://your.domain/appdir"
    ASSET_URL="${APP_URL}"
    MIX_URL="${APP_URL}"
    ```
1. Add the following in `config/app.php`, below `url`:
    ```

    'url' => env('APP_URL', 'http://localhost'),

    'asset_url' => env('ASSET_URL', '/'),

    'mix_url' => env('MIX_URL'),
   ```
1. Configure `base` in `vite.config.js`:
    ```
    ...
    export default defineConfig({
        base: '/appdir/build',
        plugins: [
            ...
        ],
    });
    ```

This works for me with both Laravel 10.x and 11.x.
